src/client/src/assets/moleculer.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
```

Comment:
For accessibility, could you add the following attributes:

* `alt` brief description of this image
* `aria-hidden="true"` so screen readers ignore this image

And the following element in the body of the `<svg>`:

```xml
<svg>
    <title>Moleculer Logo</title>
    ...
</svg>
```

Reply:
Done https://github.com/microsoft/WebTemplateStudio/pull/1173/commits/5e962fa4222cdd7036a40ef014e5714bb1e8c246